sherlock/internal/models/role.go

```diff
@@ -169,6 +171,13 @@ func (r *Role) AfterCreate(tx *gorm.DB) error {
 		To:        r.RoleFields,
 	}).Error; err != nil {
 		return fmt.Errorf("failed to create RoleOperation: %w", err)
+	} else {
+		slack.SendPermissionChangeNotification(tx.Statement.Context, user.SlackReference(true), slack.PermissionChangeNotificationInputs{
+			Summary: fmt.Sprintf("created Role %s", *r.Name),
+			Results: []string{
+				"Fields: " + slack.EscapeText(litter.Sdump(r.RoleFields)),
```

Comment:
Correct, yeah, `litter` imports `go-spew` under the hood, but from poking around it seems consensus is actually to use `litter` now because it is actively maintained (even if it still has a reference to `go-spew`)